["```py\n0011001010001001000101001001000100100010000010101\n0010101010101000000111111110010010100111110100100\n0100100010010100101011101111011010101010101010101\n\n010100000100000101010100\n\n10100100100100010101000101001001010101001001001001\n00011110101011010110100101011100000000000000000111\n```", "```py\n01010000 01000001 01010100\n```", "```py\nfrom ctypes import string_at\nfrom sys import getsizeof\nfrom binascii import hexlify\n\na = 0b01010000_01000001_01010100\nprint(a)\n>>> 5259604\n\n# prints out the memory of the variable\nprint(hexlify(string_at(id(a), getsizeof(a))))\n>>> b'0100000000000000607c054995550000010000000000000054415000'\n\ntext = \"PAT\"\nprint(hexlify(string_at(id(text), getsizeof(text))))\n>>>b'0100000000000000a00f0649955500000300000000000000375c9f1f02'\n   b'acdbe4e5379218b77f0000000000000000000050415400'\n```", "```py\n>>>import datetime\n>>>datetime.datetime.now()\ndatetime.datetime(2020, 9, 8, 22, 19, 28, 838667)\n```", "```py\ndef close_kitchen_if_past_cutoff_time(point_in_time):\n    if point_in_time >= closing_time():\n        close_kitchen()\n        log_time_closed(point_in_time)\n```", "```py\ndef close_kitchen_if_past_cutoff_time(point_in_time: datetime.datetime):\n    if point_in_time >= closing_time():\n        close_kitchen()\n        log_time_closed(point_in_time)\n```", "```py\n>>>[] + {}\nTypeError: can only concatenate list (not \"dict\") to list\n\n>>> {} + []\nTypeError: unsupported operand type(s) for +: 'dict' and list\n```", "```py\n>>> [] + {}\n\"[object Object]\"\n\n>>> {} + []\n0\n```", "```py\n>>> a = 5\n>>> a = \"string\"\n>>> a\n\"string\"\n\n>>> a = tuple()\n>>> a\n()\n```", "```py\n>>> a: int = 5\n>>> a = \"string\"\n>>> a\n\"string\"\n```", "```py\nfrom typing import Iterable\ndef print_items(items: Iterable):\n    for item in items:\n        print(item)\n\nprint_items([1,2,3])\nprint_items({4, 5, 6})\nprint_items({\"A\": 1, \"B\": 2, \"C\": 3})\n```", "```py\n>>> print_items(5)\n\nTraceback (most recent call last):\n  File \"<stdin>\", line 1, in <module>\n  File \"<stdin>\", line 2, in print_items\nTypeError: 'int' object is not iterable\n\n>>> '__iter__' in dir(int)\nFalse\n>>> '__iter__' in dir(list)\nTrue\n```", "```py\n>>>def double_value(value):\n>>>    return value + value\n\n>>>double_value(5)\n10\n\n>>>double_value(\"abc\")\n\"abcabc\"\n```", "```py\n>>>double_value([1, 2, 3])\n[1, 2, 3, 1, 2, 3]\n```"]